lib/DDG/Meta/Data.pm

```diff
@@ -122,6 +123,37 @@ sub get_ia {
     return $m;
 }
 
+# filter_ias({ repo => 'goodies', dev_milestone => 'live'... })
+# Lookups combine as an AND operation.
+# Returns a list of IAs on `wantarray', otherwise an (id => ia) HASH ref.
+#
+# Each condition consists of a $key and $lookup.
+# $lookup should be either a string, ARRAY ref, or CODE ref.
+# If a string, then $lookup is compared with `eq'
+# If a CODE ref, then $lookup is called with the IAs $key attribute and
+# should return a boolean value.
+# If an ARRAY ref, then the above two rules are used with each element, the
+# IA only needs to satisfy one.
+sub filter_ias {
+	my ($lookups) = @_;
```

Comment:
Uhh, why is this package a 'class'? It doesn't have anything we'd want to expose? Surely using it as a package with an export so people can do `use DDG::Meta::Data qw(filter_ias by_id ...)` would be more useful?

I guess this might break something though, so... Meh.